input/pages/changes.md

```diff
@@ -8,7 +8,15 @@
 - Addition of support for GenomicStudy to capture structed, study level metadata. This is back porting a resource introduced in R5 as a set of profiles, code systems, and value sets into this R4 based IG. This includes structured extensions to capture region studied details, and the existing profile was removed. See [here](StructureDefinition-genomic-study.html) for more information.
 - Broadening support for biomarker observations by introducing a new, more generic MolecularBiomarker profile, along with refactoring existing profiles and examples. Explore the new profile [here](StructureDefinition-molecular-biomarker.html).
 - Improved guidance on the Somatic Reporting page, including a detailed synthetic report and additional example instances. See the new page [here](somatics.html).
-- Update in the representation of molecular consequences through the creation of a new Molecular Consequences observation profile that is treated as an Implication. This is a non-passive change, as several Variant(Observation).component and DiagnosticImplication(Observation).component slices were moved to the new profile. See [here](StructureDefinition-molecular-consequence.html) for the new profile.
+- Update in the representation of molecular consequences through the creation of a new Molecular Consequences observation profile that is treated as an Implication. This is a non-passive change, as several Variant(Observation).component and DiagnosticImplication(Observation).component slices were moved to the new profile. See [here](StructureDefinition-molecular-consequence.html) for the new profile. A list of changes includes: 
+  * Variant
+    * rename component:coding-hgvs to component:representative-coding-hgvs (same LOINC code)
```

Comment:
Did we provide any guidance on what we mean by 'representative'? I suppose it is fairly obvious, but perhaps we should write up a quick comment about it? (and not here, probably in the Variant profile).

Reply:
I added some guidance into the Intro section of the Variant profile.